src/openrct2/Game.cpp

```diff
@@ -544,7 +549,12 @@ void save_game_cmd(const utf8* name /* = nullptr */)
 {
     if (name == nullptr)
     {
-        save_game_with_name(gScenarioSavePath.c_str());
+        char savePath[MAX_PATH];
```

Comment:
This is possible to be reached if you open an existing sv6 save file and use the console to `save_park` it will save with the wrong extension.